app/views/dashboard/_distribution_widget.html.erb

```diff
@@ -1,3 +1,4 @@
+<!-- TODO: Replace with either just out of sync or origin specific interval -->
```

Comment:
this should be solved the same way as in `DashboardHelper#count_reports` since it's the same thing